playwright_test/page/chatwidget.page.js

```diff
@@ -0,0 +1,40 @@
+import { expect } from "@playwright/test";
+import { URL_DETAILS } from "../utils/Url";
+import { APP_ID } from "../utils/Constant";
+
+class ChatWidgetPage {
+    constructor(page) {
+        this.page = page;
+
+        //Locators
+        this.testRadioBtn = page.locator("#test");
+        this.appIDbox = page.getByPlaceholder("23a6d30a59d247fa64605501f58074ea3");
+        this.launchWidgetBtn = page.getByRole("button", {name: "Launch Widget"});
+        this.kommunicateChatWidgetiframe = page.frameLocator('iframe[name="Kommunicate widget iframe"]');
+        this.chatWidgetLaunchBtn = this.kommunicateChatWidgetiframe.locator("#launcher-svg-container");
+        this.chatWidgetTextBox = this.kommunicateChatWidgetiframe.locator("#mck-text-box");
+        this.chatWidgetMsgSendBtn = this.kommunicateChatWidgetiframe.locator("#mck-msg-sbmt");
+        this.chatWidgetDeliveredIcon = this.kommunicateChatWidgetiframe.locator('xpath=//span[@title="delivered"]');
+    }
+
+    //Launch chat widget
+    async launchChatWidget() {
+        await this.page.goto(URL_DETAILS.CHATWIDGET_URL);
+        await this.testRadioBtn.click();
+        await this.appIDbox.click();
+        await this.appIDbox.press("Meta+a");
```

Comment:
what is Meta+a ?

Reply:
Meta+a is used to do Select all (Control+A) action.